style.css

```diff
@@ -1,7 +1,122 @@
-body div:last-of-type {
-  border: 1px solid blue;
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 0.625rem;
+}
+
+body {
+  background-color: rgba(5, 84, 202, 0.815);
+  padding: 2rem;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.header h1 {
+  font-family: "Montserrat", sans-serif;
```

Comment:
- [ ] The `font-family: "Montserrat", sans-serif;` has been repeated severally in your code. Please ensure you assign an extra universal class in your HTML file where the style applies and use that one class in your css file. This makes your code DRY and easy to debug.
eg.
 ```
monterrat-font {
   font-family: "Montserrat", sans-serif;
}
```

- [ ] Please ensure you apply these changes to the `font-weight: 200;`, `border: 1px solid orangered;` which have also been repeated.